add dest to scoreboard

Huskydog9988 opened the pull request:
fixes #835

Huskydog9988 committed:
add dest to scoreboard

Huskydog9988 committed:
fix board and action bar keys for dest display

Huskydog9988 committed:
fix color on action bar

okx-code commented on plugins/railswitch-paper/src/main/java/sh/okx/railswitch/settings/SettingsManager.java:
```diff
@@ -27,10 +38,15 @@ public static void init(RailSwitchPlugin plugin) {
         destSetting = new DestinationSetting(plugin);
         resetSetting = new ResetSetting(plugin, destSetting);
+        destDisplayLocation = new DisplayLocationSetting(plugin, DisplayLocationSetting.DisplayLocation.SIDEBAR,
+            "Dest Display Location", "destDisplayLocation", new ItemStack(Material.ARROW), "the  status");
```
"the  status"?

okx-code commented on plugins/railswitch-paper/src/main/java/sh/okx/railswitch/settings/SettingsManager.java:
```diff
@@ -27,10 +38,15 @@ public static void init(RailSwitchPlugin plugin) {
         // Register those elements
         menu.registerToParentMenu();
         menu.registerSetting(destSetting);
         menu.registerSetting(resetSetting);
```
destDisplayLocation is not being registered

okx-code reviewed (changes requested):
Destination display setting doesn't work

Huskydog9988 committed:
fix dest display setting

okx-code reviewed (changes requested):
When I change the destination setting visibility it doesn't actually disappear from the sidebar? Did you test this?

Huskydog9988 committed:
Update player dest display if setting changes

Also fixed issue where registered listeners would be called before new value is set

Huskydog9988 committed:
switch destSetting updating UI to be a listener

okx-code reviewed (changes requested):
When I relog the destination disappears from the sidebar

Huskydog9988 committed:
resend dest display on connect
